src/runner/runESLint.js

```diff
@@ -17,14 +35,7 @@ const runESLint = ({ testPath, config, extraOptions }) => {
     require(config.setupTestFrameworkScriptFile);
   }
 
-  const { cliOptions: baseCliOptions } = getESLintOptions(config);
-  const cliOptions = {
-    ...baseCliOptions,
-    fix: getComputedFixValue(baseCliOptions),
-    ...extraOptions,
-  };
-
-  const cli = new CLIEngine(cliOptions);
+  const { cli, formatter, cliOptions } = getCachedValues(config, extraOptions);
```

Comment:
what happens if runESLint is called with a different config/extraOptions when cached values exist? Is that worth checking for?

Reply:
Possibly, but I'm not sure realistically if this would ever happen, since I don't think you can modify the config while it is running. Maybe watch mode? Even then, I'm not sure it would be worth the overhead.